scripts/views/mapboxView.js

```diff
@@ -314,6 +369,19 @@
     toggleFilters('selected-border', filter);
   };
 
+  // cleares all selections and district visibility - used in toggle
+  mapboxView.removeSelections = function()  {
+    map.setLayoutProperty('states-house-districts-selected', 'visibility', 'none');
+    map.setLayoutProperty('states-senate-districts-selected', 'visibility', 'none');
+
+    map.setLayoutProperty('states-house-districts', 'visibility', 'none');
+    map.setLayoutProperty('states-senate-districts', 'visibility', 'none');
+
```

Comment:
the district outlines are already handled by the button toggle, this function should only be removing the selected states, not the outlines.